scripts/download/tools.mjs

```diff
@@ -118,7 +118,7 @@ export default async function main(platform) {
     entryName:        `${ kubePlatform }-amd64/${ exeName('helm') }`,
   });
 
-  // Download Kim
+  // Download KimImageProcessor
```

Comment:
Why is Kim renamed to KimImageProcessor?

Reply:
I did an interactive global search-and-replace, but didn't skip that one.